UsbAccess/UsbAccess.cpp

```diff
@@ -702,5 +706,52 @@ namespace Plugin {
 
         return result;
     }
+
+    uint32_t UsbAccess::getUSBSizeWrapper(const JsonObject &parameters, JsonObject &response)
```

Comment:
Pls add support for multiple USB drives.

Reply:
Will check if this is supported on a reference product and add details.